website/utils/url_utils.ts

```diff
@@ -13,23 +13,38 @@ const LUT_REGEX = /^[a-z0-9.]*:[ ]*[a-z0-9.]*$/;
 const HEX_COLOR_REGEX = /^[0-9a-fA-F]{6}$/;
 
 export type ViewerChannelSettingJson = {
-  /** Color, as a 6-digit hex color */
+  /** Color, as a 6-digit hex color.  */
   col?: string;
-  /** Colorize. "1" is enabled. */
+  /** Colorize. "1" is enabled. Disabled by default. */
   clz?: "1" | "0";
-  /** Colorize alpha, in the [0, 1] range. */
+  /** Colorize alpha, in the [0, 1] range. Set to `1.0` by default. */
   cza?: string;
-  /** Opacity, in the [0, 1 range] */
+  /** Isosurface alpha, in the [0, 1 range]. Set to `1.0` by default.*/
   isa?: string;
   /** LUT to map from intensity to opacity. Should be two alphanumeric values separated
-   * by a colon, e.g. "0:255", "p50:p90", or "autoij:0".
+   * by a colon. The first value is the minimum and the second is the maximum.
+   * Defaults to [0, 255].
+   *
+   * - Plain numbers are treated as direct intensity values.
+   * - `p{n}` represents a percentile, where n is a percentile in the [0, 100].
+   * - `m{n}` represents the median multiplied by n.
```

Comment:
i think it's really multiplying the median intensity by n/100 (so that the code didn't have to deal with decimals)

Reply:
I just noticed this is also documented in ViewerChannelSetting too.